var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -36,6 +37,9 @@ class NeurodamusCore(Package):
     depends_on('py-enum34',        type=('run',), when='^python@2.4:2.7.999,3.1:3.3.999')
     depends_on('py-lazy-property', type=('run'), when='+python')
 
+    def get_git_fetcher(self):
+        return next((fetcher for fetcher in self.fetcher if (isinstance(fetcher, GitFetchStrategy) and fetcher.url == self.git)), None)
```

Comment:
From https://github.com/BlueBrain/spack/blob/develop/lib/spack/spack/package.py#L665 I understand it is guaranteed the root fetcher to be the first one in the list. Maybe we could just check it's git. 
That also simplifies the line (a very nice 1-liner though!) :)
Something like this? 

```suggestion
        root_fetcher = self.fetcher[0]
        return root_fetcher if isinstance(root_fetcher, GitFetchStrategy) else None
```

Reply:
I like that, a lot simpler!

Reply:
I agree with you, I just added the other checks just to be sure